react.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
 
         "react/jsx-indent-props": [2, 4],
 
-        "react/jsx-space-before-closing": 2,
+        "react/jsx-tag-spacing": 2,
```

Comment:
The rules aren't in alphabetical order at all :) can create a separate PR afterwards?